### PR TITLE
feat: Deprecation of RMRK

### LINKED ISSUE #10643: Deprecation of RMRK
### Task

1. Remove logic related to rmrk and ksm prefixes

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10643

TODO:
1. - [x] search endpoint: `https://polysearch.w.kodadot.xyz/search`
 context: https://github.com/kodadot/polysearch/blob/main/supabase/functions/search/index.ts
solution: search collection according to current urlPrefix

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/carousel/module/CarouselInfo.vue [hunk 1]
```diff
@@ -84,5 +84,5 @@ const showPrice = computed((): boolean => {
   return Number(price.value) > 0 && !isCollection
 })
 
-const unitSymbol = computed(() => prefixToToken[props.item.chain || 'ksm'])
+const unitSymbol = computed(() => prefixToToken[props.item.chain || 'ahp'])
```
Comment (vikiival) [inline components/carousel/module/CarouselInfo.vue:87]:
```suggestion
const unitSymbol = computed(() => prefixToToken[props.item.chain || 'dot'])
```

File: components/teleport/Teleport.vue [hunk 2]
```diff
@@ -338,16 +338,6 @@ const isDisabled = (chain: Chain) => {
 }
 
 const fromNetworks = [
```
Comment (vikiival) [inline components/teleport/Teleport.vue:340]: why

File: utils/config/chain.config.ts [hunk 3]
```diff
@@ -33,7 +33,7 @@ export const chainAssetOf = (prefix: Prefix): AssetItem => {
 }
 
 export const defultTokenChain: Record<string, Prefix> = {
-  KSM: 'ksm',
+  KSM: 'ahk',
```
Comment (vikiival) [inline utils/config/chain.config.ts:36]:
```suggestion
  KSM: 'ksm',
```

**Review by vikiival — CHANGES_REQUESTED** [review]
Keep:
- teleport, send

**Comment by Jarsen136** [comment]
> Keep:
> 
> * teleport, send

well, if so, we have to keep some code/config for the `ksm`/`rmrk` network. I will try to make it back.

File: components/carousel/module/CarouselInfo.vue (hunk 1 above)
Comment (Jarsen136) [reply components/carousel/module/CarouselInfo.vue:87]: I can't see the deficiencies here as they all return `DOT`
image

**Comment by Jarsen136** [comment]
> > Keep:
> > 
> > * teleport, send
> 
> well, if so, we have to keep some code/config for the `ksm`/`rmrk` network. I will try to make it back.

✅ done

File: components/balance/MultipleBalances.vue [hunk 4]
```diff
@@ -86,12 +86,7 @@ import { formatNumber } from '@/utils/format/balance'
 import { NeoButton, NeoSkeleton } from '@kodadot1/brick'
 import { ChainToken, type ChainType, useIdentityStore } from '@/stores/identity'
 
-const displayChainOrder: ChainType[] = [
-  'polkadot',
-  'polkadotHub',
-  'kusama',
-  'kusamaHub',
-]
+const displayChainOrder: ChainType[] = ['polkadot', 'polkadotHub', 'kusamaHub']
```
Comment (vikiival) [inline components/balance/MultipleBalances.vue:89]: ahp, ahk, polkadot, kusama

File: components/common/ConnectWallet/WalletAssetMenu.vue [hunk 5]
```diff
@@ -78,11 +78,6 @@ const menus = ref<{ label: string; to: string; vm: ChainVM[] }[]>([
     to: `/${urlPrefix.value}/teleport`,
     vm: ['SUB'],
   },
-  {
-    label: 'Migrate',
-    to: '/migrate',
-    vm: ['SUB'],
-  },
```
Comment (vikiival) [inline components/common/ConnectWallet/WalletAssetMenu.vue:85]: would keep migrator. 
cc @damskyftw ?

Comment (Jarsen136) [reply components/common/ConnectWallet/WalletAssetMenu.vue:85]: Really? If we keep the migrate section, we have to maintain the indexer for rmrk/ksm and keep it runing.

Comment (vikiival) [reply components/common/ConnectWallet/WalletAssetMenu.vue:85]: Would just hide the route

**Comment by Jarsen136** [comment]
continue to https://github.com/kodadot/nft-gallery/pull/10686

**Pull request closed without merging**